cmd/ticommunityowners/main.go

```diff
@@ -67,8 +67,6 @@ func main() {
 	}
 
 	logrus.SetFormatter(&logrus.JSONFormatter{})
```

Comment:
Can we also set the formatter in load func?

Reply:
Maybe we can use https://github.com/kubernetes/test-infra/blob/aa40187f73e1395fa2145d78ee43b03a2c59666b/prow/logrusutil/logrusutil.go#L55.

Reply:
You can refer to https://github.com/kubernetes/test-infra/blob/aa40187f73e1395fa2145d78ee43b03a2c59666b/prow/external-plugins/refresh/main.go#L89.

Reply:
If you use JSONFormatter uniformly, it is indeed possible.

Reply:
Maybe we can just use logrusutil.

Reply:
The default setting of `logrusutil.ComponentInit()` is `PrintLineNumber: true`, the output information will be more specific, do we need this information?
```
{"client":"github","file":"D:/GoProject/pkg/mod/k8s.io/test-infra@v0.0.0-20210223071603-952586a1435a/prow/github/client.go:705","func":"k8s.io/test-infra/prow/github.(*client).log","level":"info","msg":"Throttle(360, 360)","time":"2021-03-02T12:58:06+08:00"}
```

Reply:
I think it's better.